pkgs/tools/security/pcsclite/default.nix

```
@@ -10,6 +10,7 @@
 , polkit
 , systemdMinimal
 , IOKit
+, polkitSupport ? false
 }:
 
 stdenv.mkDerivation rec {
```

```suggestion
stdenv.mkDerivation rec {
  pname = "pcsclite" + lib.optionalString "-with-polkit";
```
or we leave this out entirely which I would prefer since people tend to use pname wherever they find a string matching it.

---

I think you mean

```suggestion
  pname = "pcsclite" + lib.optionalString polkitSupport "-with-polkit";
```

---

> or we leave this out entirely which I would prefer since people tend to use pname wherever they find a string matching it.

This is necessary for nix-env -q/nix search not to list (apparently) duplicated packages: https://discourse.nixos.org/t/types-of-package-duplication-in-nix-search/6631/3